### PR TITLE
Add PowerShell snippet to disable toll free dial

### PR DESCRIPTION
**Pull request by cchiffersMSFT**
Adding in a powershell snipped to disable toll free dialin.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/disabling-toll-free-numbers-for-specific-teams-users.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ From the **Microsoft Teams admin center**:
 > [!Note]
 > [!INCLUDE [updating-admin-interfaces](includes/updating-admin-interfaces.md)]
  
-**Using PowerShell**  
+**Using PowerShell** 
 
-See the [Microsoft Teams PowerShell reference](/powershell/module/teams/?view=teams-ps) for more information.
\ No newline at end of file
+You can disable toll-free dial in for a specific user using PowerShell:
```
Comment (JohanFreelancer9) [inline Teams/disabling-toll-free-numbers-for-specific-teams-users.md:61]:
```suggestion
You can disable toll-free dial-in for a specific user using PowerShell:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cchiffersMSFT, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
This is a very old PR and it has branch conflicts.  Closing....  If this is still an issue, it can be resubmitted.

**Pull request closed without merging**